### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by JauntAR**
This PR sets up the master-react-admin structure.

- Remove unused modules from master-react
- Remove unused pages from master-react
- Change LandingPage to MapIndex.tsx (update the style configs)
- Create single appfile call 'reactmaster-admin' for now
- Update the viewar.config => "appId": "com.viewar.reactmaster-admin"
- Uninstall viewar-core package

### CODE REVIEW

**Comment by JauntAR** [comment]
@kh-viewar When you have time you can review this, I will pause on it for a bit.

File: package.json [hunk 1]
```diff
@@ -107,6 +109,7 @@
     "lodash.throttle": "^4.1.1",
     "md5": "^2.2.1",
     "normalize.css": "^8.0.1",
+    "npm": "^6.14.8",
```
Comment (kh-viewar) [inline package.json:112]: Do we need "npm" and "install" packages in here? Looks like an accident happened in bash.

File: src/components/Patterns/MapItem/MapItem.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </FormGroup>
+    </div>
+  );
+}
+
+MapItem.propTypes = {
+  activeStatus: PropTypes.bool,
+  id: PropTypes.number,
+  name: PropTypes.string,
+}
```
Comment (kh-viewar) [inline src/components/Patterns/MapItem/MapItem.tsx:66]: I'd prefer this if we go for the TS version, but the current .propTypes is fine too.
```js
interface MapItemType = {
  activeStatus?: bool,
  id?: number,
  name?: string
}

const MapItem = React.FC<MapItemType>({activeStatus, id, name}) {}
```

File: src/index.html.ejs [hunk 3]
```diff
@@ -21,7 +21,6 @@
       window.bundleIdentifier = window.bundleIdentifier || <%= JSON.stringify(htmlWebpackPlugin.options.bundleIdentifier) %>
       window.bundleVersion = window.bundleVersion || <%= JSON.stringify(htmlWebpackPlugin.options.bundleVersion) %>
     </script>
-    <script src="./viewar-core.js"></script>
```
Comment (kh-viewar) [inline src/index.html.ejs:24]: I think we need this import to run it with `npm run start`

File: src/style.default.js [hunk 4]
```diff
@@ -1,15 +1,26 @@
...
+      color: "#000",
+    },
+  },
+  MapImportStyles: {
+    Background: require("assets/images/landing_background.jpg"),
+    Logo: require("assets/images/ViewAR_Logo_White_Lightred.png"),
+    ImportButton: {
+      backgroundColor: "#ED220E",
+      border: "2px solid #000",
+      color: "#ffffff",
```
Comment (kh-viewar) [inline src/style.default.js:22]: I think we can get rid of the styling through a configurable JSON at all. But let's leave it right now.

**Review by kh-viewar — COMMENTED** [review]
I have some more comments about the general structure, but let's discuss it in person.

File: package.json (hunk 1 above)
Comment (JauntAR) [reply package.json:112]: I don't know how this got in there, not the first time... will remove.

File: src/components/Patterns/MapItem/MapItem.tsx (hunk 2 above)
Comment (JauntAR) [reply src/components/Patterns/MapItem/MapItem.tsx:66]: I prefer to stick to typescript as much as possible from now on. Will update (btw we use `type` in some places and `interface` in others?)

File: src/index.html.ejs (hunk 3 above)
Comment (JauntAR) [reply src/index.html.ejs:24]: Will add this back

File: src/style.default.js (hunk 4 above)
Comment (JauntAR) [reply src/style.default.js:22]: Ok let's leave it.

File: src/components/Patterns/MapItem/MapItem.tsx (hunk 2 above)
Comment (kh-viewar) [reply src/components/Patterns/MapItem/MapItem.tsx:66]: I think it's no problem to mix it. Or do you think that's bad?

Comment (JauntAR) [reply src/components/Patterns/MapItem/MapItem.tsx:66]: I was just wondering what the difference is (if you knew)

Comment (JauntAR) [reply src/components/Patterns/MapItem/MapItem.tsx:66]: https://pawelgrzybek.com/typescript-interface-vs-type/#:~:text=Interface%20declarations%20can%20exclusively%20represent,makes%20the%20type%20more%20flexible.

Comment (kh-viewar) [reply src/components/Patterns/MapItem/MapItem.tsx:66]: with InferProps we generate TS typing information out of propTypes, with React.FC we create propTypes out of TS typing information. With the type information we can reuse them more easily in ts files + it's easier to type imo (`id: string` vs `id: PropTypes.string.isRequired`).

**Pull request merged**